tests/test_functional.py

```diff
@@ -76,6 +76,8 @@
     ('get_html', '/langdoc/status', None, None),
     ('get_html', '/langdoc/status/browser?macroarea=Eurasia', None, None),
     ('get_html', '/langdoc/status/languages-1-3?macroarea=Eurasia&year=2018&family=', None, None),
+    # Blueprint Tests
+    ('get', '/bp/api/search?bpsearch=anglai', None, '[{"glottocode": "stan1293", "iso": "eng", "name": "English", "level": "language"}, {"glottocode": "midd1317", "iso": "enm", "name": "Middle English", "level": "language"}, {"glottocode": "tsha1245", "iso": "tsj", "name": "Tshangla", "level": "language"}]'),
```

Comment:
Oh I forgot, you should refactor this test out into a separate function. And add additional test cases for the different control flows in `bpsearch`